### DISCUSSION TITLE
ret_dict having issue with pandas

### DISCUSSION THREAD

**Discussion by Pheoxis** [trackmania-rl/tmrl#69] [Ideas]
I would like to raise an issue with pandas. If you are having an issue with ret_dict and that it can't convert {x} to numeric then its propably connected with your pandas verision. The newer veriosn of pandas don't co-operate with some functions. Now I am using pandas==2.0.3. So if you having this issue install this veriosn of pandas and try to use ".detach().item()" in ret_dict.

raise TypeError(f"Could not convert {x} to numeric")
TypeError: Could not convert [tensor(-2.0272, device='cuda:0') tensor(0.0926, device='cuda:0')
 tensor(-110.2940, device='cuda:0')] to numeric

**Comment by yannbouteiller** [comment]
Hi, the last release of `tmrl` should (hopefully) comply with `pandas` expectations?
